mbuild/tests/test_lammpsdata.py

```diff
@@ -63,65 +63,72 @@ def test_save_charmm(self):
     @pytest.mark.skipif(not has_foyer, reason="Foyer package not installed")
     def test_charmm_improper(self):
         import mbuild as mb
```

Comment:
Can we put all the imports at the top of the function here, please.

Reply:
Sure, but I had it like it was because the pre-existing `test_save_charmm` was like that.  I'll fix that too while I'm at it.